osu.Server.Spectator/Services/SharedInterop.cs

```diff
@@ -140,17 +167,22 @@ public async Task<long> CreateRoomAsync(int hostUserId, MultiplayerRoom room)
             })));
         }
 
-        public async Task AddUserToRoomAsync(int userId, long roomId, string password)
+        public Task AddUserToRoomAsync(int userId, long roomId, string password)
```

Comment:
Minor note, no action expected: Feels a smidge weird to have this with an `Async` suffix and returning `Task` if this method is just gonna be a synchronous fire-and-forget now. Like I wouldn't expect an async method to fire-and-forget.

Reply:
I think in general "-Async" is for the consumer not the implementation, because removing the `async` keyword indicates no functional difference to the consumer.